common/src/main/java/com/flipkart/flux/persistence/CryptHashGenerator.java

```diff
@@ -0,0 +1,22 @@
+package com.flipkart.flux.persistence;
+
+import java.security.MessageDigest;
+
+/**
+ * Created by amitkumar.o on 06/06/17.
+ */
+public class CryptHashGenerator {
+
+    private static final String cryptHashAlgorithmPrefix = "SHA-256";
+
+    public static Character getUniformCryptHash(String stateMachineId) {
+        try {
+            MessageDigest md = MessageDigest.getInstance(cryptHashAlgorithmPrefix);
+            md.update(stateMachineId.getBytes());
+            String cryptHash = javax.xml.bind.DatatypeConverter.printHexBinary(md.digest()).toLowerCase();
+            return cryptHash.charAt(0);
+        } catch (Exception ex) {
+            throw new RuntimeException();
```

Comment:
empty runtime exception... difficult to figure out why it's failed. Wrap the actual exception with runtime exception.